librosa/feature/spectral.py

```diff
@@ -1674,7 +1682,14 @@ def mfcc(y=None, sr=22050, S=None, n_mfcc=20, dct_type=2, norm='ortho', **kwargs
     if S is None:
         S = power_to_db(melspectrogram(y=y, sr=sr, **kwargs))
 
-    return scipy.fftpack.dct(S, axis=0, type=dct_type, norm=norm)[:n_mfcc]
+    M = scipy.fftpack.dct(S, axis=0, type=dct_type, norm=norm)[:n_mfcc]
+
+    if lifter > 0:
+        M *= 1 + (lifter / 2) * np.sin(np.pi * np.arange(1, 1 + n_mfcc) / lifter)[:, np.newaxis]
+    elif lifter < 0:
```

Comment:
note that `lifter=np.nan` will return `M` unchanged. That's undocumented, untested, and probably not what the user will intend.

Reply:
hmmm, good catch.  I'll fix it.